sdk/textanalytics/azure-ai-textanalytics/tests/test_analyze_sentiment_async.py

```diff
@@ -698,7 +698,7 @@ async def test_opinion_mining_more_than_5_documents(self, client):
     @GlobalTextAnalyticsAccountPreparer()
     @TextAnalyticsClientPreparer()
     async def test_opinion_mining_no_mined_opinions(self, client):
-        document = (await client.analyze_sentiment(documents=["today is a hot day"], show_opinion_mining=True))[0]
+        document = (await client.analyze_sentiment(documents=["today is a hot day"]))[0]
```

Comment:
i think this call should still include `show_opinion_mining=True`

Reply:
Thanks, I've updated this here and also in test_analyze_sentiment.py.  I've also updated the recordings.